styles/_use-cases.scss

```diff
@@ -30,13 +29,48 @@ $custody-logos: "gatehub", "bitgo";
     }
   }
 }
+
+.modal-content-uses {
+  .first-ledger-bot{
+    min-height: 80px !important;
+    position: relative;
+    bottom: 20px;
+    content: url("../img/uses/first-ledger-bot.svg");
+  }
+  .orchestra-finance{
+    content: url("../img/uses/orchestra-finance.svg");
+  }
+  .moai-finance{
+    min-height: 80px !important;
+    position: relative;
+    bottom: 20px;
+    content: url("../img/uses/moai-finance.svg");
+  }
+  .ledger-city{
+    margin: 0px !important;
+    position: relative;
+    bottom: 4px;
+    left: 6px;
+    max-height: 47px !important;
+  }
+  .zerpmon {
+    margin: 0px;
+    min-width: 80px;
+    min-height: 84px;
+    position: relative;
+    bottom: 13px;
+    content: url("../img/uses/zerpmon.png") !important;
+  }
+}
+
 #use_case_companies_list {
   #threezy {
     .biz-logo {
       max-height: 40px;
-      content: url("../img/uses/modallogos/threezy.png");
+      content: url("../img/uses/modallogos/threezy.png") !important;
```

Comment:
The `!important` doesn't seem to do anything. It's bad practice to use `!important` where you can avoid it.

If it's not needed, can we remove it from here and also the light theme versions of the styles?